oak_functions/logger/Cargo.toml

```diff
@@ -6,4 +6,5 @@ edition = "2021"
 license = "Apache-2.0"
 
 [dependencies]
-log = "*"
+# Logging via the log crate is disabled and only available using our own custom logger.
```

Comment:
Maybe add link to our own custom logger (trait) and add why we disable logging via log crate?

Reply:
Expanded the comment and added a link to the trait and a TODO.